src/coinify/trade.js

```diff
@@ -133,6 +133,22 @@ Object.defineProperties(CoinifyTrade.prototype, {
   }
 });
 
+function toCents (fiat) {
```

Comment:
These functions would be useful in other places too, so let's put them in coinify/helpers.js